cmd/server/main.go

```diff
@@ -75,5 +75,5 @@ func handleConn(conn tpt.CapableConn) error {
 	if _, err := str.Write([]byte(data)); err != nil {
 		return err
 	}
-	return str.Close()
+	return str.CloseWrite()
```

Comment:
ditto (that or reset)

Reply:
Hm. Actually, in this case, shouldn't it just be a close? I mean, we've already read everything we want.

Reply:
That works as well. And it's easier.